base/src/test/java/io/spine/validate/RangeConstraintTest.java

```diff
@@ -78,4 +92,13 @@ void throwOnMalformedRanges(String badRange) {
                 Arguments.of("[3...5]")
         );
     }
+
+    private static Stream<Arguments> rangeCombinationsFor(Number left, Number right) {
+        return Stream.of(
+                Arguments.of(format("[%s..%s]", left, right)),
```

Comment:
This block does not stand the modification of the enumeration of possible edges. Please refactor so that you don't have to repeat the code which is largely the same.